src/discordcr/rest.cr

```diff
@@ -52,8 +58,10 @@ module Discord
           end
 
           if response.headers["X-RateLimit-Global"]?
+            LOGGER.warn "Locking global mutex for #{retry_after}"
```

Comment:
How about just s/Locking global mutex/Pausing requests/

Reply:
I think its important for the user to know they've actually hit the global rate limit and thus all requests will be affected, vs only a single route being locked.